.github/workflows/notebook_ci.yaml

```diff
@@ -34,17 +34,37 @@ jobs:
 
     - name: Convert Notebooks
       run: | 
-        mkdir tutorials/notebooks
+        rm -rf notebooks
+        mkdir notebooks
         jupytext --to notebook tutorials/*.md
-        mv tutorials/*.ipynb tutorials/notebooks
+        mv tutorials/*.ipynb notebooks
 
     - name: Run tests
       run: |
         jupyter kernel &
-        pytest --nbmake tutorials/notebooks/*.ipynb --overwrite --nbmake-timeout=500
+        pytest --nbmake notebooks/*.ipynb --overwrite --nbmake-timeout=500
 
     - name: Create Notebook Artifact
       uses: actions/upload-artifact@v3
       with:
         name: executed-notebooks
-        path: tutorials/notebooks/*.ipynb
+        path: notebooks/*.ipynb
+
+  publish:
+    needs: pytests
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
```

Comment:
Maybe with `checkout@v3` we need something like:
```
with:
  token: ${{ secrets.GH_TOKEN }}
```
to get the right permissions. Not sure.